cmake/public/gridtools_setup_targets.cmake

```diff
@@ -228,7 +228,8 @@ macro(_gt_setup_targets _config_mode clang_cuda_mode)
         list(APPEND GT_BACKENDS cuda)
         list(APPEND GT_ICO_BACKENDS cuda)
 
-        if(MPI_CXX_FOUND)
+        option(GT_GCL_GPU "Disable if your MPI implementation is not CUDA-aware" ON)
```

Comment:
I think this option should appear when both `MPI` and `CUDA` are found or it should be always available.

Reply:
You are right. I am doing too many things at the same time :(